benchmarks/src/jmh/java/com/linecorp/armeria/grpc/downstream/DownstreamSimpleBenchmark.java

```diff
@@ -62,11 +61,11 @@ protected GithubServiceFutureStub normalFutureClient() {
 
     @Override
     protected void setUp() throws Exception {
-        server = new ServerBuilder()
-                .serviceUnder("/", new GrpcServiceBuilder().addService(new GithubApiService()).build())
-                .requestTimeout(Duration.ZERO)
-                .meterRegistry(NoopMeterRegistry.get())
-                .build();
+        server = Server.builder()
+                       .serviceUnder("/", new GrpcServiceBuilder().addService(new GithubApiService()).build())
```

Comment:
Don't we need to change for `GrpcServiceBuilder`?

Reply:
Do you mean , It should be invoking of GrpcService.builder() not invoking constructor of GrpcServiceBuilder? 
If so, I 'm planning to refactor the class next pull-request.  
If you think It is better to deal wtih the class in this pull request, I'll work on that!

Reply:
Yes, I think there's no reason to treat the `GrpcServiceBuilder` differently.  I think we can do it in this PR. 😄

Reply:
I got it. I'll deal with the class next commit. Thank you.